bos_mint/forms.py

```diff
@@ -248,6 +295,26 @@ def update(self, selectedId):
             selectedId,
             InternationalizedString.parseToList(self.name), self.sport.data)
 
+class EventGroupFormDetails(FlaskForm):
```

Comment:
In general it is not a good idea to introduce code duplication to extend logic. 

Refine existing class or introduce a derivation

Reply:
Will refine while fixing BOS-55 in next sprint